daisy_workflows/build-publish/bare_metal/rhel_8.wf.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Name": "rhel-8-metal",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "source_image_project": "bct-prod-images",
+    "build_date": "${DATE}",
+    "workflow_root": {"Value": "/workflows", "Description": "Root of github workflows, defaults to /workflows in the container."},
+    "gcs_url": {"Required": true, "Description": "The GCS path that image raw file exported to."}
+  },
+  "Steps": {
+    "build-bare-metal": {
+      "Timeout": "70m",
+      "IncludeWorkflow": {
+        "Path": "bare_metal_linux.wf.json",
```

Comment:
"Path": "./bare_metal_linux.wf.json",

Reply:
done

Reply:
This is not validate after move wf into image_build folder